jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -119,6 +120,24 @@ AC_DEFUN([OPENJ9_CONFIGURE_COMPILERS],
   AC_SUBST(OPENJ9_DEVELOPER_DIR)
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_CRIU_SUPPORT],
```

Comment:
In jdk8, changes to configuration files such as this require re-running `bash common/autoconf/autogen.sh`.